maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -60,7 +60,7 @@ class Orchestra(
     private val onCommandMetadataUpdate: (MaestroCommand, CommandMetadata) -> Unit = { _, _ -> },
 ) {
 
-    private lateinit var jsEngine: JsEngine
+    lateinit var jsEngine: JsEngine
```

Comment:
I am assuming this is done public for a reason. Do you want to use it in some other class?